test_utils/utils.go

```diff
@@ -54,6 +58,35 @@ func WipeEtcd() {
 	}
 }
 
+// GetResultForCurrent takes the session output with cniVersion and returns the Result in current.Result format.
+func GetResultForCurrent(session *gexec.Session, cniVersion string) (*current.Result, error) {
+
+	// Check if the version is older than 0.3.0.
+	// Convert it to 0.3.1 standard if that is the case.
+	if version.Compare(cniVersion, "0.3.0", "<") {
+		r020 := types020.Result{}
+
+		if err := json.Unmarshal(session.Out.Contents(), &r020); err != nil {
+			log.Fatalf("Error unmarshaling session output to Result: %v\n", err)
+		}
+
+		r030, err := current.NewResultFromResult(&r020)
```

Comment:
to represent the result is upconverted to 030 result type (filled with "zero value" for the fields not present in older spec version results)